docs/ldoc.css

```diff
@@ -679,19 +1193,31 @@ pre .url { color: #272fc2; text-decoration: underline; }
 }
 
 .summary_row {
-    background-color: white;
+    background-color: var(--summary-row-background-color);
 }
 
 .summary_row:hover {
-    background-color: #f7fbff;
+    background-color: var(--summary-row-hover-background-color);
+}
+
+/*
+.svg_stroke,
+.svg_fill {
+    color: #fff;
 }
 
-/* CSS for beautiful.fg_normal */
-.svg_stroke, .svg_fill {
-    color: black;
+svg {
+    color: #fff;
+    fill: #fff;
+    stroke: #fff;
 }
+*/
```

Comment:
Should we remove these comment blocks?

Reply:
Done. However, it will now require a new review release from your side...